CODEOWNERS

```diff
@@ -12,16 +12,16 @@
 /samples/msal-browser-samples/ @derisen @sameerag @tnorling @hectormmg @jo-arroyo @peterzenz @konstantin-msft
 
 # MSAL Common
-/lib/msal-common/ @sameerag @tnorling @hectormmg @jo-arroyo @peterzenz @bgavrilMS @robbie-microsoft @konstantin-msft
+/lib/msal-common/ @sameerag @tnorling @hectormmg @jo-arroyo @peterzenz @robbie-microsoft @konstantin-msft
```

Comment:
You may want this? Okay, @Robbie-Microsoft is there, so may be fine. But a lot of `msal-node` has code derived from common FYI.

Reply:
I know, but getting too many emails from this.